x-pack/legacy/plugins/maps/server/test_utils/index.js

```diff
@@ -37,10 +37,16 @@ export const getMockKbnServer = (
       }),
     },
     xpack_main: {},
-    task_manager: {
-      registerTaskDefinitions: () => undefined,
-      schedule: () => Promise.resolve(),
-      fetch: mockTaskFetch,
```

Comment:
The best thing to do here would be to just remove this portion as you have, and not replace it with anything. We aren't using task manager as of the recent merge of #52834. Really this whole function `getMockKbnServer` could stand to go, but for the purposes of this PR, maybe just remove `task_manager` and its value.

Reply:
Cool, removed :)